### PR TITLE
salloc, sattach, sbcast, sdiag, sprio, sshare, strigger, sview, sh5util, scrun, scrontab: add pages

### PR DESCRIPTION
**Pull request by gutjuri**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 23.11

### CODE REVIEW

File: pages/linux/sh5util.md
Comment (kbdharun) [inline pages/linux/sh5util.md:6]: Kinda curious, what does `(-step)` mean here, does it mean job step?

File: pages/linux/sh5util.md
Comment (kbdharun) [inline pages/linux/sh5util.md:3]: Is it necessary that this part should be inside brackets?

```suggestion
> Merge HDF5 files produced by the `sacct_gather_profile` plugin.
```

File: pages/linux/scrontab.md
Comment (kbdharun) [inline pages/linux/scrontab.md:6]: Should we place crontab inside backticks here or is it fine as it is?

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution. I have a few queries after which this PR is GTG.

File: pages/linux/scrontab.md
Comment (gutjuri) [reply pages/linux/scrontab.md:6]: It is not used in the sense of the command `crontab`, but in the sense of the concept of a crontab. That's why i didn't put it in backticks.

**Review by kbdharun — APPROVED** [review]

**Comment by kbdharun** [comment]
Before merging this PR, I would suggest waiting for @sbrl's review to checkout the commands, as they use Slurm in their university.

**Comment by sbrl** [comment]
Wow, this is an enormous PR! I'll check this thoroughly tomorrow, as it's 3:30am here right now.

File: pages/linux/salloc.md
Comment (sbrl) [inline pages/linux/salloc.md:3]: This doesn't really allocate nodes though? It allocates a job on a node.
```suggestion
> Start an interactive shell session or execute a command by allocating on 1 or more nodes in a cluster. 
```

File: pages/linux/sattach.md
Comment (sbrl) [inline pages/linux/sattach.md:6]: Define `Make available` here.

File: pages/linux/sattach.md
Comment (sbrl) [inline pages/linux/sattach.md:12]: What do you mean by `task number`? Is this a job number, or something else?

Try to use the official language from the project itself.

Further, what do you mean by transmit to the specified task? I'm rather confused. Is this attaching interactively, or pointing something else at a job's stdin?

File: pages/linux/sattach.md
Comment (sbrl) [inline pages/linux/sattach.md:14]: Please try to keep the language consistent. In this page alone you've used `display`, `transmit`, and `make available`. This is very confusing.

File: pages/linux/sbcast.md
Comment (sbrl) [inline pages/linux/sbcast.md:13]: Do you have a link to documentation on this? All the documentation I've seen for `sbcast` doesn't have `--send-libs` as an argument.

File: pages/linux/scrontab.md
Comment (sbrl) [inline pages/linux/scrontab.md:12]: Should we also have an example here for interactively editing the crontab of the current user?

File: pages/linux/scrontab.md
Comment (sbrl) [inline pages/linux/scrontab.md:18]: Need to be specific here given previous examples.
```suggestion
- Print the crontab of the current user to `stdout`:
```

File: pages/linux/scrun.md
Comment (sbrl) [inline pages/linux/scrun.md:12]: Is there a way to pass arguments to a container at all? It might be worth having an extra example directly beneath this one about that if so.

File: pages/linux/scrun.md
Comment (sbrl) [inline pages/linux/scrun.md:8]: What do yo  mean by create a container here? Are we creating it from nothing? Where does the image come from? Is this like `docker build`, or does it download the container from somewhere?

It would be nice if this example was specific about where the container comes from.

File: pages/linux/sdiag.md
Comment (sbrl) [inline pages/linux/sdiag.md:6]: Again with the accuracy and specificity of language here.
```suggestion
- Show all performance counters related to the execution of `slurmctld`:
```

File: pages/linux/sprio.md
Comment (sbrl) [inline pages/linux/sprio.md:6]: Grammar
```suggestion
- View the factors determining the scheduling priority of all jobs:
```

File: pages/linux/sprio.md
Comment (sbrl) [inline pages/linux/sprio.md:16]: `--user` and `--long` should be 2 separate examples.

File: pages/linux/strigger.md
Comment (sbrl) [inline pages/linux/strigger.md:3]: An extract description line of context would be useful here:
```suggestion
> View or modify Slurm trigger information.
> Triggers are actions that are automatically run when a given event occurs on a Slurm cluster.
```

File: pages/linux/strigger.md
Comment (sbrl) [inline pages/linux/strigger.md:12]: Can you pass arguments too? If so, might be worth an extra example here.

**Review by sbrl — CHANGES_REQUESTED** [review]
Wow, thanks for the new pages @gutjuri! I've left some comments below for you to review.

Sorry there are a lot of comments here - you're contributing a lot of pages at once, so nautrally there will be more review comments :-)

File: pages/linux/sattach.md
Comment (gutjuri) [reply pages/linux/sattach.md:6]:
```suggestion
-  Redirect the IO streams (`stdout`, `stderr`, and `stdin`) of a Slurm job step to the current terminal:
```

Do you think this would be better?

File: pages/linux/sattach.md
Comment (gutjuri) [reply pages/linux/sattach.md:12]: "Task" is an official term. I cannot find a document providing an exact definition, but the term is mentioned several times in the docs, e.g. here https://slurm.schedmd.com/cpu_management.html#Section2

See also: https://stackoverflow.com/questions/39186698/what-does-the-ntasks-or-n-tasks-does-in-slurm

File: pages/linux/sattach.md
Comment (gutjuri) [inline pages/linux/sattach.md:10]:
```suggestion
- Use the current console's input as `stdin` to the specified task:
```

Not sure how to formulate this better.

File: pages/linux/sattach.md
Comment (gutjuri) [reply pages/linux/sattach.md:14]:
```suggestion
- Only redirect `stdin`/`stderr` of the specified task:
```

File: pages/linux/sbcast.md
Comment (gutjuri) [reply pages/linux/sbcast.md:13]: https://slurm.schedmd.com/sbcast.html#SECTION_OPTIONS

File: pages/linux/scrontab.md
Comment (gutjuri) [reply pages/linux/scrontab.md:12]: You're right, i'll add one.

Comment (gutjuri) [inline pages/linux/scrontab.md:12]:
```suggestion
- [e]dit the crontab of the current user:

`scrontab -e`

- [e]dit the crontab of the specified user:

`scrontab --user={{user_id}} -e`
```

File: pages/linux/scrun.md
Comment (gutjuri) [reply pages/linux/scrun.md:8]: Looks like I misunderstood `scrun`. It seems like scrun is not meant as an interface to users (https://slurm.schedmd.com/SLUG23/containers-slug23.pdf, slide 14), but for the developers of e.g. docker, podman etc (?). 

For me, this raises the question on whether to include `scrun` in tldr at all.

File: pages/linux/sprio.md
Comment (gutjuri) [inline pages/linux/sprio.md:16]:
```suggestion
- Output additional information:

`sprio --long`

- View information for the jobs of specified users:

`sprio --user={{user_name_1,user_name_2,...}}`
```

File: pages/linux/strigger.md
Comment (gutjuri) [reply pages/linux/strigger.md:13]: The docs state

1. "A hostlist expression for the nodelist or job ID is passed as an argument to the program."
2. "You may quote the path and include extra program arguments if desired."

So I think that you could do

```suggestion
`strigger --set --jobid={{job_id}} --fini --program="{{path/to/executable}} {{argument1 argument2 ...}}"`
```

Do you think that that's correct?

**Comment by gutjuri** [comment]
> Wow, thanks for the new pages @gutjuri! I've left some comments below for you to review.
> 
> Sorry there are a lot of comments here - you're contributing a lot of pages at once, so nautrally there will be more review comments :-)

Thanks for your detailed feedback. I think I have addressed all of the issues you raised in my comments. Could you check out my suggestions and comment or add a thumbs up if you think they're fine?

File: pages/linux/sattach.md
Comment (sbrl) [reply pages/linux/sattach.md:6]: Yes, that looks much better @gutjuri. Many thanks!

File: pages/linux/sattach.md
Comment (sbrl) [reply pages/linux/sattach.md:12]: Hmm okay, that makes sense. Thanks for the clarification.

Could you explain what you mean by `transmit to the specified task` though please?

File: pages/linux/scrun.md
Comment (sbrl) [reply pages/linux/scrun.md:8]: Hmmm, interesting.

I don't see why it shouldn't be included, even if it isn't user-facing @gutjuri. But I would suggest it isn't a priority on including it vs eg. `sbatch` for example.

Your call.

File: pages/linux/strigger.md
Comment (sbrl) [reply pages/linux/strigger.md:13]: That seems good. Let's do that, @gutjuri.

File: pages/linux/sattach.md
Comment (sbrl) [reply pages/linux/sattach.md:10]: That suggestion reads much better - let's do that.

**Comment by sbrl** [comment]
Looks good, thanks for the hard work @gutjuri! Got a few more comments there for you. I think we're quite close!

File: pages/linux/sattach.md
Comment (gutjuri) [reply pages/linux/sattach.md:12]: Basically that I send input from the current console to the specified task.

File: pages/linux/scrun.md
Comment (gutjuri) [reply pages/linux/scrun.md:8]: I'll remove the page then and hope someone with experience using this command will create it.

**Review by sbrl — APPROVED** [review]
Looks good to me! Thanks so much for your hard work on this, @gutjuri! Large PRs like this was that add lots of pages can be a challenge - thanks for persevering with the long review process!

**Pull request merged**
